### PR TITLE
update localization

### PR DESCRIPTION
**Pull request by gtalha07**
This PR updates the localized content of the app.

### CODE REVIEW

**Comment by sachintha-s** [comment]
Looks good to me

**Review by sachintha-s — APPROVED** [review]

File: app/lib/l10n/app_de.arb [hunk 1]
```diff
@@ -1,35 +1,53 @@
...
-    "password": "Passwort",
-    "passwordValidate": "Passwort kann nicht leer sein",
-    "loginSuccess": "Anmeldung erfolgreich",
-    "loginFailed": "Anmeldung fehlgeschlagen",
-    "noName":"kein Name",
-    "noAddr": "keine Adresse",
-    "loadingName": "Ladename"
+    "select": "Auswählen",
+    "photo":"Foto",
+    "file":"datei",
```
Comment (gnunicorn) [inline app/lib/l10n/app_de.arb:6]:
```suggestion
    "file":"Datei",
```

File: app/lib/l10n/app_de.arb [hunk 2]
```diff
@@ -1,35 +1,53 @@
...
+    "members":"Mitglieder",
+    "loadingConvo":"Konversationen werden geladen",
+    "emptyPlaceholderText": "Hier sieht es leer aus",
+    "message": "Nachricht",
+    "loadingName":"Name wird geladen",
+    "noName":"Kein Name",
+    "toDoList":"Aufgabenliste",
```
Comment (gnunicorn) [inline app/lib/l10n/app_de.arb:13]:
```suggestion
    "toDoList":"Aufgaben",
```

File: app/lib/l10n/app_de.arb [hunk 3]
```diff
@@ -1,35 +1,53 @@
...
+    "gallery":"Galerie",
+    "events":"Veranstaltungen",
+    "sharedResource":"Freigegebene Ressource",
+    "pollsVotes":"Umfragen & Abstimmungen",
+    "groupBudgeting":"Gruppenbudgetierung",
```
Comment (gnunicorn) [inline app/lib/l10n/app_de.arb:18]:
```suggestion
    "groupBudgeting":"CoBudget",
```

**Review by gnunicorn — APPROVED** [review]
Just some german fixes

**Pull request merged**